2_process/src/process_data.R

```diff
@@ -0,0 +1,43 @@
+process_data <- function(file_in){
+  # Prepare the data for plotting
+  readr::read_csv(mendota_file, col_types = 'iccd') %>%
```

Comment:
This won't actually work because you are trying to read the path from `mendota_file`, but your function argument is named `file_in`. The function will only know about the objects you pass it!

Reply:
duhh oops